lib/Model/ScheduledMessageMapper.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\Talk\Model;
+
+use OCA\Talk\Exceptions\InvalidRoomException;
+use OCA\Talk\Room;
+use OCP\AppFramework\Db\DoesNotExistException;
+use OCP\AppFramework\Db\QBMapper;
+use OCP\AppFramework\Db\TTransactional;
+use OCP\DB\QueryBuilder\IQueryBuilder;
+use OCP\IDBConnection;
+
+/**
+ * @method ScheduledMessage mapRowToEntity(array $row)
+ * @method ScheduledMessage findEntity(IQueryBuilder $query)
+ * @method list<ScheduledMessage> findEntities(IQueryBuilder $query)
+ * @method ScheduledMessage update(ScheduledMessage $scheduledMessage)
+ * @method ScheduledMessage delete(ScheduledMessage $scheduledMessage)
+ * @template-extends QBMapper<ScheduledMessage>
+ */
+class ScheduledMessageMapper extends QBMapper {
+	use TTransactional;
+
+	public function __construct(
+		IDBConnection $db,
+	) {
+		parent::__construct($db, 'talk_scheduled_msg', ScheduledMessage::class);
```

Comment:
```suggestion
		parent::__construct($db, 'talk_scheduled_messages', ScheduledMessage::class);
```

- Should even work before https://github.com/nextcloud/server/pull/51593